app/controllers/dashboard/cards_controller.rb

```diff
@@ -38,9 +39,52 @@ def destroy
   end
 
   def get_flickr_images
-    @flickr_images = FlickrService.search_photos(params[:search],
-                                                 '1, 2, 3, 4, 5, 6, 7',
-                                                 'url_sq, url_m', 10, 1)
+    @flickr_images = FlickrService.search_photos(
+      # A free text search. Photos who's title, description or tags contain the
+      # text will be returned. You can exclude results that match a term by
+      # prepending it with a - character.
+      params[:search],
+
+      # The license id for photos (for possible values see the
+      # flickr.photos.licenses.getInfo method). Multiple licenses may be
+      # comma-separated.
+      '1, 2, 3, 4, 5, 6, 7',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.